services/movie_session.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()

```

Comment:
```suggestion
    movie_sessions = MovieSession.objects.all()
    if session_date:
        movie_sessions = movie_sessions.filter(show_time__date=session_date)
    return movie_sessions
```
In future this code easy support